src/frontends/tensorflow_common/include/helper_ops/string_constant.hpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "internal_operation.hpp"
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+
+/// Pseudo-entity for storing strings
+class StringConstant : public InternalOperation {
```

Comment:
Let us inherit this class from `UnsupportedConstant`. So there will be no need to update `ConstToResultRemover`